docs/quantities.rst

```diff
@@ -0,0 +1,228 @@
+============================
+Quantities: Values and Units
+============================
+
+The PVL specifications supports the notion that PVL Value Expressions
+can contain an optional PVL Units Expression that follows the PVL Value.
+
+By default, the ``pvl`` library includes the :class:`pvl._collections.Units`
+class which is implemented as a :class:`collections.namedtuple` with
+a ``value`` and a ``unit`` parameter.  This kind of Python object, although
+not supported by the standard library, is sometimes also referred to as a
```

Comment:
I don't understand this comment, as namedtuple is part of the standard library?

Reply:
Ah, yeah, I can see how that is confusing.  Surely, `collections.namedtuple` is part of the standard library, and our `pvl._collections.Units` is a subclass of `collections.namedtuple`.

What I wanted to get across is that there is no 'quantity' object in the Python standard library.  There is no fundamental object type that represents a value and the units of that value.  However, libraries like astropy and pint have implemented 'quantity' objects (and managed to name them both Quantity, but they have slightly different interfaces).  In a way, the `pvl._collections.Units` is a poor version of a 'quantity' object.  It doesn't have any special powers (like the astropy and pint objects), but it is a single object which represents both a value and the units of that value.

Does this make more sense?  If so, I'll update the documentation.

Reply:
makes sense, thanks!

Reply:
Okay, fixed that up.